packages/nimble-components/src/table/models/keyboard-navigation-manager.ts

```diff
@@ -215,12 +216,6 @@ implements Subscriber {
         }
     }
 
-    public handleFocusedCellRecycling(hadRowOrCellFocus: boolean): void {
-        if (hadRowOrCellFocus && !this.focusWithinTable) {
-            this.focusCurrentRow(false);
```

Comment:
Note: The only case where we hit this code path was when an anchor was focused, and then a table scroll happened. Then, at the start of `table.handleFocusedCellRecycling();` focus was within the table, but it no longer was when this method was called (due to the anchor being blurred by its `focusedRecycleCallback`).

Now that the `focusedRecycleCallback` is gone, the code `handleChange()` which refocuses the cell does this. So this method goes away.